README.md

```diff
@@ -54,10 +54,7 @@ Add a new configuration with the created module
 config/test.exs
 
 ```elixir
-  config :xcribe,
-    information_source: YourAppWeb.Support.DocInformation,
-    format: :swagger,
-    output: "app_doc.json"
+  config :xcribe, YourAppWeb.Endpoint, information_source: YourAppWeb.Support.DocInformation
```

Comment:
@Danielwsx64 Wouldn't make sense to ditch the `information_source` completely and have all the information directly here in the config?

Reply:
I agree with that @thiamsantos. But I'll remove it on next PRs, when I intent to add a new way to add descriptions to the routes (probably using an `.exs` file as we discusses once on a slack thread).